tictactoe.rb

```diff
@@ -1,184 +1,185 @@
-class Game
-
-	def initialize
-		random_start = rand(2)
-		if random_start == 0
-			@player_turn = "X"
-		else
-			@player_turn = "O"
+	class Game
+
+		def initialize
+			random_start = rand(2)
+			if random_start == 0
+				@player_turn = "X"
+			else
+				@player_turn = "O"
+			end
 		end
-	end
-	attr_reader :player_turn
+		attr_reader :player_turn
 
-	def turn
-		puts "#{@player_turn}'s turn."
-		puts "Enter your move: "
-		
+		def turn
+			puts "#{@player_turn}'s turn."
+			puts "Enter your move: "
+			
 
-	end
+		end
 
-	def switch_turn
-		if @player_turn == "X"
-			@player_turn = "O"
-		elsif @player_turn == "O"
-			@player_turn = "X"
+		def switch_turn
+			if @player_turn == "X"
+				@player_turn = "O"
+			elsif @player_turn == "O"
+				@player_turn = "X"
+			end
 		end
-	end
 
-	def start_game
-		puts "Welcome to Tic Tac Toe"
-		puts "Play by typing the column position and row position separated by a space"
-		puts "Ex:  top middle"
-		puts "\"#{@player_turn}\" goes first."
+		def start_game
+			puts "Welcome to Tic Tac Toe"
+			puts "Play by typing the column position and row position separated by a space"
+			puts "Ex:  top middle"
+			puts "\"#{@player_turn}\" goes first."
 
+		end
 	end
-end
 
-class Board
+	class Board
```

Comment:
Your classes should be in separate files

Reply:
is that just to make the code more readable? The hard part is getting variables shared between classes